src/components/JsonView.vue

```diff
@@ -12,11 +14,16 @@
 
   const props = defineProps<{
     value?: string | null | undefined,
+    showLineNumbers?: boolean,
   }>()
 
-  const innerHtml = computed(() => {
-    return highlight(props.value ?? '', languages.json, 'json')
+  const lines = computed(() => {
+    return highlight(props.value ?? '', languages.json, 'json').split('\n')
   })
+
+  const classes = computed(() => ({
```

Comment:
does this need to be a computed?

Reply:
Probably unlikely but I wouldn't want developers to pass in a reactive boolean and not get the result they expected.